src/request/src/cache.js

```diff
@@ -70,9 +68,9 @@ export default class CacheRequest extends Request {
 
   set url(urlString) {
     super.url = urlString;
-    let pathname = global.decodeURIComponent(url.parse(urlString).pathname);
-    const urlParts = pathname.replace(/^\//g, '').split('/');
-    // "pathname" has the following form "/namespace/appKey/collection/id"
+    const pathname = global.decodeURIComponent(url.parse(urlString).pathname);
```

Comment:
@georgiwe I don't quite remember why `escape()` was used before. It could be possible that the `urlString` was not escaped before and therefore needed be but now it is by `url.parse()` although I am just speculating.

Regardless this change LGTM too.